### PR TITLE
Update admin_ceph_upgrade.xml

### PR DESCRIPTION
**Pull request by p-se**
Add step to remove previous version of Node Exporter which would prevent
a successful deployment of the containerized Node-Exporter.

Signed-off-by: Patrick Seidensal <pseidensal@suse.com>

### CODE REVIEW

File: xml/admin_ceph_upgrade.xml [hunk 1]
```diff
@@ -609,6 +609,13 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
      </para>
     </tip>
    </step>
+   <step>
+    <para>
+     Remove the Node-Exporter. It does not need to be migrated and
```
Comment (asettle) [inline xml/admin_ceph_upgrade.xml:614]:
```suggestion
     Remove the node exporter. It does not need to be migrated and

File: xml/admin_ceph_upgrade.xml [hunk 2]
```diff
@@ -609,6 +609,13 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
...
+     will be installed as a container when the <literal>specs.yaml</literal> file is applied.
```
Comment (asettle) [inline xml/admin_ceph_upgrade.xml:615]:
```suggestion
     will be installed as a container when the <filename>specs.yaml</filename> file is applied.

**Review by asettle — COMMENTED** [review]
Small nits :)

File: xml/admin_ceph_upgrade.xml [hunk 3]
```diff
@@ -609,6 +609,13 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
      </para>
     </tip>
    </step>
+   <step>
+    <para>
+     Remove the node exporter. It does not need to be migrated and
+     will be installed as a container when the <filename>specs.yaml</filename> file is applied.
+    </para>
+<screen>&prompt.cephuser;zypper rm golang-github-prometheus-node_exporter</screen>
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:617]:
```suggestion
<screen>&prompt.sudo;zypper rm golang-github-prometheus-node_exporter</screen>
```

File: xml/admin_ceph_upgrade.xml [hunk 4]
```diff
@@ -609,6 +609,13 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
      </para>
     </tip>
    </step>
+   <step>
+    <para>
+     Remove the node exporter. It does not need to be migrated and
+     will be installed as a container when the <filename>specs.yaml</filename> file is applied.
```
Comment (lproven) [inline xml/admin_ceph_upgrade.xml:615]: Would "re-installed as a container" be equally true and perhaps a little more reassuring?

**Review by asettle — APPROVED** [review]

**Review by lproven — APPROVED** [review]

**Review by lproven — APPROVED** [review]

**Pull request merged**